PROJECT-BRIEF.md

```diff
@@ -1,59 +1,186 @@
+| Optional         | Office Hours                                                                                                                                                      |
+
+### Week 7, {DATES_WEEK_7}
+
+| 🤝 Developer pairings | 🖥 Assigned Github issue                                                                 |
+| --------------------- | ---------------------------------------------------------------------------------------- |
+| Full team             | [14. As a user, I want the app to look professional and welcoming]({REPO_URL}/issues/14) |
+
+| 📖 Learning modules                                                                | Description                                                                                                 |
+| ---------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------- |
+| [How to rock the technical interview](https://www.youtube.com/watch?v=X-7eKmYqM8M) | This video is full of great insights that will help you be successful in a vairety of tehcnical interviews. |
+| [Pair interview practice](./learning-modules/pair-interview-practice.md)           | Get to know these questions so that you can pair up with a fellow Collabie next week and practice.          |
+
+| 👋 Sync meetings | Agenda                                                                                                                                                            |
+| ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Weekly Sync      | - Demo <br> - Retro <br> - Next week's issues                                                                                                                     |
+| Pair Programming | Schedule a pairing session(s) with your partner early in the week to complete the development work required to meet the acceptance criteria of your Github issue. |
+| Optional         | - Office Hours <br> - Design meeting with Mark                                                                                                                    |
+
+### Week 8, {DATES_WEEK_8}
+
+| 🤝 Developer pairings | 🖥 Assigned Github issue                                                                |
+| --------------------- | --------------------------------------------------------------------------------------- |
+|                       | No assigned issues. This is a good chance to address any emergent work defined by team. |
+
+| 📖 Learning modules                                                               |
+| --------------------------------------------------------------------------------- |
+| No learning modules this week. A great time to catch up on any outstanding tasks! |
```

Comment:
Wild that we have a free week now! That'll be nice for folks, probably!

Reply:
Right!? I was surprised to see it!